src/services/token.service.js

```diff
@@ -0,0 +1,27 @@
+import { Token } from "../models/token.js";
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId }});
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
+}
+
+function getByToken(refreshToken) {
+  return Token.findOne({ where: { refreshToken } });
+}
+
+function remove(userId) {
+  return Token.destroy({ where: { userId } })
```

Comment:
Consider adding error handling for the `remove` function to handle cases where the token might not be found or the operation fails.